source/compatibility.txt

```diff
@@ -0,0 +1,153 @@
+by the various versions of the {+driver-long+}.
+
+The first column lists the driver versions.
+
+.. include:: /includes/compatibility-table-legend-ruby.rst
+
+.. include:: /includes/language-compatibility-table-ruby.rst
+
+The driver does not support older versions of Ruby.
+
+Rails/ActiveSupport Compatibility
+---------------------------------
+
+The {+driver-short+} does not depend on ActiveSupport. However, if your
+application uses ActiveSupport or Ruby on Rails, you must load the driver's
+ActiveSupport compatibility code for behavior such as time serialization to be
+correct:
+
+.. code-block:: ruby
+
+   require 'mongo'
+   require 'mongo/active_support'
+
+Applications using Mongoid 7.0.6 or newer do not need to explicitly load
+the driver's ActiveSupport code, since Mongoid automatically does so.
+
+.. _tls-compatibility:
+
+TLS/SSL Compatibility
+----------------------
+
+The {+driver-short+} uses the protocols supported by the underlying Ruby
+``openssl`` extension. The ``openssl`` extension generally exposes
+the functionality available in the operating system's OpenSSL library.
+
+Industry best practices, and some regulations, require the use of TLS 1.1
+or newer. Some operating systems or versions might not provide an OpenSSL version
+that supports these TLS versions.
+
+If you use macOS older than 10.13 (High Sierra), you need to install Ruby from
+`rvm`_, `homebrew`_, `macports`_, or another similar source. See
+`Installing Ruby`_ for more options.
+
+If you use Linux or other non-macOS Unix systems, you can check your OpenSSL version
+as follows:
+
+.. code-block:: sh
+
+   openssl version
+
+If the version number is less than 1.0.1, support for TLS 1.1 or newer is
+not available. Contact your operating system vendor for a solution or upgrade
+to a newer distribution.
+
+You can check your TLS version by running the following command in your Ruby
+interpreter:
+
+.. code-block:: sh
+
+   ruby -e "require 'net/http'; require 'json'; puts JSON.parse(Net::HTTP.get(URI('https://www.howsmyssl.com/a/check')))['tls_version']"
+
+After running the command, you must see ``TLS 1.X`` where ``X`` is greater than
+or equal to ``1``.
+
+To learn more about TLS versions and their security implications, see `Transport Layer Security Cheat Sheet
+<https://cheatsheetseries.owasp.org/cheatsheets/Transport_Layer_Security_Cheat_Sheet.html>`_.
+
+.. _rvm: https://rvm.io/
+.. _homebrew: https://brew.sh/
+.. _macports: https://www.macports.org/
+.. _Installing Ruby: https://www.ruby-lang.org/en/documentation/installation
+
+Atlas Compatibility
+-------------------
+
+We recommend {+driver-short+} version 2.6.1 or later when using MongoDB Atlas.
+This version has significant performance improvements for TLS connections, and
+all Atlas connections use TLS.
+
+When running on JRuby and connecting to Atlas Free Tier, {+driver-short+}
+version 2.6.4 or later and Java 8 or later are required.
```

Comment:
The 2.6 series of the driver is quite old, and I don't think we want to be recommending it in any way. For example, version 2.6.4 (the final version of the 2.6 series) was released on 22 Jan 2019 -- almost 6 years ago. That's long enough ago that we probably don't even need to worry about mentioning these "performance improvements".

Ideally, we could remove this section entirely. If it needs to remain, I'd recommend at least changing the wording so that we "recommend" the latest driver version.

Reply:
I'm good with removing 👍

Reply:
Agreed, let's remove it